### DISCUSSION TITLE
Could We Add wkhtmltopdf to Wolfi?

### DISCUSSION THREAD

**Discussion by Naveen-is-here** [wolfi-dev/os#29692] [Q&A] [answered]
Hi Wolfi Team,

 I’m currently working on a project that needs wkhtmltopdf for converting HTML into PDFs, but I noticed it’s not available in the Wolfi repositories.

I think adding this package would be super beneficial for many developers who are looking to utilize this functionality. It would make life a lot easier for folks like me who rely on it!

I’m not exactly sure who to reach out to regarding this, so any help or direction you could provide would be greatly appreciated. Thanks a lot for considering this, and I’m looking forward to hearing from you!

Best,
Naveen

**Comment by xnox** [comment, marked as answer]
Do you mean https://github.com/wkhtmltopdf/wkhtmltopdf ?

> This repository has been archived by the owner on Jan 2, 2023. It is now read-only.

Typically in Wolfi we only have supported software by upstream only.

**Reply by Naveen-is-here** [reply]
Hi @xnox,

Thanks for the response!

Since `wkhtmltopdf` is archived, is there any package or a list of packages in Wolfi that I could use for HTML to PDF conversion, like Chromium, Puppeteer, or something similar?

Thanks!  
Naveen
